CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ option(JSON_FORMAT "Build JSON formatter" "OFF")
 option(CATA_CCACHE "Try to find and build with ccache" "ON")
 option(CATA_CLANG_TIDY_PLUGIN "Build Cata's custom clang-tidy checks as a plugin" "OFF")
 option(CATA_CLANG_TIDY_EXECUTABLE "Build Cata's custom clang-tidy checks as an executable" "OFF")
-option(TESTS "Compile Cata's tests" "OFF")
+option(TESTS "Compile Cata's tests" "ON")
```

Comment:
@akrieger this should fix the GCC7 job which seems to assume CMake by default builds the test. The build of test is anyway disabled from using the presets.